### PR TITLE
Add libcbor

### PR DESCRIPTION
**Pull request by kelvinhammond**

### CODE REVIEW

**Comment by kozross** [comment]
Sorry about the delay @kelvinhammond - happy to merge once you resolve the conflict.

**Comment by kozross** [comment]
@kelvinhammond Mind resolving the conflicts? I can merge once you do.

**Comment by kelvinhammond** [comment]
@kozross updated.

File: README.md [hunk 1]
```diff
@@ -871,6 +871,7 @@ This includes libraries for things like XML, JSON, CSV, and other similar format
 ### Others ###
 
 * [libbson][235] - BSON utility library. [``Apache-2.0``][Apache-2.0]
+* [libcbor][573]: CBOR protocol implementation for C and others
```
Comment (kozross) [inline README.md:874]: Can you add a link to the MIT license here, as per the other entries, @kelvinhammond ?

Comment (kelvinhammond) [reply README.md]: updated, thanks.

**Comment by kozross** [comment]
Thanks for the contribution!

**Pull request merged**